content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -192,20 +192,23 @@ The following flags can be used:
 You can choose more than one authorization module. Modules are checked in order
 so an earlier module has higher priority to allow or deny a request.
 
-## Privilege escalation via pod creation
+## Privilege escalation via workload creation or edits {#privilege-escalation}
 
-Users who have the ability to create pods in a namespace can potentially
-escalate their privileges within that namespace.  They can create pods that
-access their privileges within that namespace. They can create pods that access
-secrets the user cannot themselves read, or that run under a service account
-with different/greater permissions.
+Users who have the ability to create or edit anything which creates pods in a
```

Comment:
This isn't quite right either. Editing a Pod or a pod template, yes, definitely potential for privilege escalation.
Editing the CRD for an operator (that indirectly creates Pods) or a similar controller? There's potential but
it's rather less likely (because that kind of controller might prevent escalation by its design). It'd be good to be precise.

Reply:
Agreed, will be more explicit about which changes can be used for the escalation

Reply:
@sftim  I've added clarifications about why each resource is problematic and a bit more explanation of the CRD part. Mind re-reviewing?

Reply:
What about?

> Users who can create/edit pods in a namespace, could escalate their privileges in that namespace. 
> This includes indirectly creating/editing pods, such as through an operator or similar controller.

Reply:
This makes it less clear that it's any controller which is what I misunderstood from the original wording. Would this work?

```
Users who can create/edit pods in a namespace, either directly or through a controller such as an operator, could escalate their privileges in that namespace.
```

Reply:
Great!